files/benchmark_job.yml

```diff
@@ -124,10 +128,26 @@
                                 compiler="http://10.40.0.13/toolchains/${file}"
                         fi
 
+                        if [ -d "${WORKSPACE}/mr-provisioner-client" ]; then
+                            rm -rf "${WORKSPACE}/mr-provisioner-client"
+                        fi
+                        git clone https://github.com/Linaro/mr-provisioner-client.git "${WORKSPACE}/mr-provisioner-client"
```

Comment:
Can't we use SCM here? It would be a lot cleaner and we wouldn't have to worry about the directory existing or anything, as Jenkins takes care of that, I think.

Reply:
You have to tell it wether to wipe it.
My main issue is that Jenkins doesn't seem to do it the "tidy" way, i.e. it git clones all the files in the workspace, with no directory to keep them all together.
There might be an option, but there certainly isn't that much documentation...

Reply:
It's ok to tell it to wipe, what I mean is that we don't need logic in the shell to do that.
Also, being in the workspace is perfectly fine, we don't need it to be in any further sub-dir.
The workspace is supposed to be cleaned before execution anyway (if that needs a special Jenkins flag, we need to set it for all jobs).

Reply:
The main issue I failed to bring up, was that the branch name in the scm is predetermined when the job is generated on the Jenkins.
Runtime branch definition seems compromised, and Google seems to point to [using the "Jenkins Pipeline"](https://cleverbuilder.com/notes/jenkins-dynamic-git-branch/), and switching to this is outside of the scope of this PR.

Reply:
Fair enough. We can do that in a following PR.